frontend/src/app/core/services/user-settings.service.ts

```diff
@@ -6,7 +6,7 @@ import { User } from '@shared/models/user/user';
 })
 export class UserSettingsService {
 
-  url = '/settings/user';
+  url = '/users';
```

Comment:
Merge User, UserSettings and HeaderImage services into one UserService

Reply:
done